vendor: bump Pebble to d5bc42c2884b

bananabrick opened the pull request:
d5bc42c2 db: add Pebble interface for CPU slot granter
478b9750 sstable: fix test suffix collector's handling of range keys
cb41b17a Replace `github.com/codahale/hdrhistogram` with `github.com/HdrHistogram/hdrhistogram-go`

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/82093)
<!-- Reviewable:end -->

bananabrick committed:
vendor: bump Pebble to d5bc42c2884b

d5bc42c2 db: add Pebble interface for CPU slot granter
478b9750 sstable: fix test suffix collector's handling of range keys
cb41b17a Replace `github.com/codahale/hdrhistogram` with `github.com/HdrHistogram/hdrhistogram-go`

Release note: None

nicktrav reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 5 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/82093#-:-N3RCjTR3ELkQDFso4-A:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @bananabrick)



<!-- Sent from Reviewable.io -->

bananabrick commented:
TFTR!

bors r+
